subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_log

```diff
@@ -58,6 +59,12 @@ config NRF_CLOUD_LOG_OUTPUT_LEVEL
 	  Log messages with a log level larger than this will be ignored.
 	  Can be overridden at runtime from the cloud.
 
+config NRF_CLOUD_LOG_INCLUDE_LEVEL_0
```

Comment:
Instead of a new KConfig, can `NRF_CLOUD_LOG_OUTPUT_LEVEL` be adjusted so that `0` is usable and `5` disables it? (or perhaps `-1`, but `5` would make intuitive sense)

Reply:
No, the level numbers are a standard part of Zephyr's logging system.  I don't want to confuse things by making the level value of `NRF_CLOUD_LOG_OUTPUT_LEVEL` have a special case.

Reply:
I think my suggestion would be aligned with Zephyr's logging system numbers, though, right? 0 is the log level used for printk, isn't it?

Reply:
You want to define a new value, 5 or -1.  Those are special cases.

Reply:
Ah, right, sorry. I was thinking of the priority levels backwards for a minute